src/map/packets/char_update.cpp

```diff
@@ -30,119 +30,342 @@
 #include "ai/ai_container.h"
 #include "ai/states/death_state.h"
 #include "entities/charentity.h"
+#include "item_container.h"
 #include "status_effect_container.h"
 #include "utils/itemutils.h"
 
+// https://github.com/atom0s/XiPackets/tree/main/world/server/0x0037
+
+// Namespace to avoid compilation units using the same structs twice with different definitions
+namespace charUpateFlags
+{
+    struct flags0_t
+    {
+        uint32_t HideFlag : 1;
+        uint32_t SleepFlag : 1;
+        uint32_t GroundFlag : 1;
+        uint32_t CliPosInitFlag : 1;
+        uint32_t LfgFlag : 1;
+        uint32_t AnonymousFlag : 1;
+        uint32_t CfhFlag : 1;
+        uint32_t AwayFlag : 1;
+        uint32_t Gender : 1;
+        uint32_t unknown_1_9 : 1;
+        uint32_t unknown_1_10 : 1;
+        uint32_t GraphSize : 2;
+        uint32_t Chocobo_Index : 3;
+        uint32_t hpp : 8;
+        uint32_t PlayOnelineFlag : 1;
```

Comment:
should we leave their typo and comment that or should we adjust it? lol

Reply:
TBH I think it's a bad idea to leave this kind of thing as is. Whatever SE did internally is irrelevant to the project, and in the future if someone tries to "fix" this it will have to be explained to them that "if you decompile the client you'll find this is actually a real typo." In the context of LSB, it just makes things more confusing and harder to search.

Reply:
@cocosolos I can see us changing it, then someone else looking at the debug symbols it came form and then trying to "fix it" to match that again. I don't think it at all matters which we use we'll just want to have an explanation visible either way to head of repeatedly explaining in chat or repeatedly swapping back and forth.